--- a/any_proxy.go
+++ b/any_proxy.go
@@ -51,7 +51,6 @@ import (
     "net"
     "os"
     "os/signal"
-    "path/filepath"
     "runtime"
     "runtime/pprof"
     "strconv"
@@ -130,10 +129,9 @@ type directorFunc func(*net.IP) bool
 var director func(*net.IP) (bool, int)
 
 func init() {
-    dir, _ := filepath.Abs(filepath.Dir(os.Args[0]))
-    gConfFile = dir + "/any_proxy.conf"
-    gLogfile = dir + "/any_proxy.log"
-    gStatsFile = dir + "/any_proxy.stats"
+    gConfFile = "/etc/any-proxy.conf"
+    gLogfile = "/dev/stdout"
+    gStatsFile = "/dev/null"
 
     flag.Usage = func() {
         fmt.Fprintf(os.Stdout, "%s\n\n", versionString())