From ab123704e73325b00a2a775916750285cdce5e4a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Thalheim?= <Mic92@users.noreply.github.com>
Date: Thu, 17 Sep 2020 16:12:10 +0100
Subject: [PATCH] fix incorrectly spelled variable

fixes #37
---
 direnvrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/direnvrc b/direnvrc
index 092bc3e..b8fbf94 100644
--- a/direnvrc
+++ b/direnvrc
@@ -30,7 +30,7 @@ _nix_import_env() {
   fi
 
   export PATH=$PATH${old_path:+":"}$old_path
-  _nix_export_or_unset TEMPDIR "$old_tmp"
+  _nix_export_or_unset TEMPDIR "$old_tmpdir"
   _nix_export_or_unset TERM "$old_term"
 
   # misleading since we are in an impure shell now
